### ISSUE TITLE
Notify users of input validation with aria-live

### ISSUE THREAD

**Issue by shainanigans** [StackExchange/Stacks#1053]
**Describe the bug**
For the [input validation](https://stackoverflow.design/product/components/inputs/#validation-examples), there should be an `aria-live="assertive"` on the validation message in cases where the message is added to the DOM/view. (There’s an example of this [here from Deque](https://dequeuniversity.com/class/dynamic-updates/notify-users/aria-live).) With this attribute, screen readers will read this announcement to users when it appears. There is no mention of this in the Stacks documentation.

**To Reproduce**
1. In Pickles (or any other SO4T instance) go to Questions, open the filter, and save a custom filter.
2. Use a screen reader of your choosing to observe what happens when you change a filter and try to save another custom filter with the same name. 
3. Note that the validation message is not read out.

**Expected behavior**
Include a note in [Inputs > Validation states ](https://stackoverflow.design/product/components/inputs/#validation-states)about using `aria-live`.
